examples/lib/stories/camera_and_viewport/fixed_resolution_example.dart

```diff
@@ -12,32 +13,84 @@ class FixedResolutionExample extends FlameGame
     It is useful when you want the visible part of the game to be the same on
     all devices no matter the actual screen size of the device.
     Resize the window or change device orientation to see the difference.
+    
+    If you tap once you will set the zoom to 2 and if you tap again it goes back
+    to 1, so that you can test how it works with a zoom level.
   ''';
 
-  FixedResolutionExample({
-    required Vector2 viewportResolution,
-  }) : super(
-          camera: CameraComponent.withFixedResolution(
-            width: viewportResolution.x,
-            height: viewportResolution.y,
-          ),
+  FixedResolutionExample()
+      : super(
+          camera: CameraComponent.withFixedResolution(width: 600, height: 1024),
+          world: FixedResolutionWorld(),
         );
 
   @override
   Future<void> onLoad() async {
-    final flameSprite = await loadSprite('layers/player.png');
+    final textRenderer = TextPaint(
+      style: TextStyle(fontSize: 25, color: BasicPalette.black.color),
+    );
+    camera.viewport.add(
+      TextButton(
+        text: 'Viewport\ncomponent',
+        position: Vector2.all(10),
+        textRenderer: textRenderer,
+      ),
+    );
+    camera.viewfinder.add(
+      TextButton(
+        text: 'Viewfinder\ncomponent',
+        textRenderer: textRenderer,
+        position: Vector2(0, 200),
+        anchor: Anchor.center,
+      ),
+    );
+    camera.viewport.add(
+      TextButton(
+        text: 'Viewport\ncomponent',
+        position: camera.viewport.virtualSize - Vector2.all(10),
```

Comment:
is this basically always 600x1024 (for example?)

Reply:
Yeah, now it is, it was configurable before but since it is now using `.controlled` it doesn't update.